.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+**What does this PR tweak:**
```

Comment:
Could you change this to
```
**What does this PR do:**
Include a small to medium description of what your PR changes. Feel free to give it a little flourish if you want to show off a new feature more completely.
```
and completely remove the "Why is this better:" section?